Vagrantfile

```diff
@@ -57,7 +42,8 @@ Vagrant.configure("2") do |config|
 		stage.vm.box = "dummy"
 		stage.vm.box_url = "https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box"
 		stage.vm.provision "file", source: ".", destination: "$HOME/devopsloft"
-		stage.vm.provision :ansible do |ansible|
+		stage.vm.provision :ansible_local do |ansible|
+			ansible.provisioning_path = "/../home/ubuntu/devopsloft"
```

Comment:
did you try to use $HOME/devopsloft instead of a hardcoded path?

Reply:
yes, but it does not work, look at the output:
```
==> stage: Running provisioner: file...
    stage: C:/devopsloft/. => /home/ubuntu/devopsloft
==> stage: Running provisioner: ansible_local...
`playbook` does not exist on the guest: /$HOME/devopsloft/playbooks/bootstrap-infra.yml
```

Reply:
Checking it myself

Reply:
ok. I understand why it is not working with $HOME. It it working with the following path: "/home/ubuntu/devopsloft"